client/src/canvas/Viewport.res

```diff
@@ -48,7 +48,7 @@ let moveRight = (m: model): modification => moveCanvasBy(m, Defaults.moveSize, 0
   less volatile method for the constant definitions.
 */
 let centerCanvasOn = (tl: toplevel): Pos.t => {
-  let windowWidth = Native.Window.viewportWidth
+  let windowWidth = Webapi.Dom.window->Webapi.Dom.Window.innerWidth
```

Comment:
```suggestion
  let windowWidth = Webapi.Dom.Window.innerWidth(Webapi.Dom.window)
```
In F#, we generally use `fn(arg)` syntax unless we need more than 1 pipe - should that same rule apply in ReScript?
In below usages of `innerWidth`, we follow that rule.

Reply:
The reason I did it the other way here, is to mimic the actual semantics under the hood, which is the JavaScript call `window.innerWidth`